R/rng.R

```diff
@@ -64,17 +68,25 @@ dust_rng <- R6::R6Class(
                                     private$float)
       private$n_generators <- n_generators
 
+      if (real_type == "float") {
+        size_int_bits <- 32L
+        name <- "xoshiro128starstar"
+      } else {
+        size_int_bits <- 64L
+        name <- "xoshiro128starstar"
```

Comment:
Just checking these are the right choices. I might have expected the second one to be 256 bits

Reply:
indeed this is wrong